src/Helper/Utils.php

```diff
@@ -27,4 +28,26 @@ public static function fillObject($object, array $properties): void
             }
         }
     }
+
+    /**
+     * @param  array $array
+     * @param  array $requiredKeys
+     *
+     * @return array $emptyValues
+     */
+    public static function emptyValues($array, $requiredKeys = null): array
```

Comment:
Ik snap dat dit lekker kort is, maar ik vind dit dus geen duidelijke naam, misschien iets in de richting van:

- `getKeysWithoutValue`
- `getEmptyValueKeys`

etc...

Vergeet niet de namen van de test en de bijbehorende provider bij te werken